src/monika/Assignment7_Shop.java

```diff
@@ -0,0 +1,122 @@
+package monika;
+/*
+ Assignment  7  : 30th March 2021 
+
+1. Create a system (Shop) which maintains each item's quantity.
+
+a. Initial stocks of Maggie: 50 Maggie packets 
+b. Initial stocks of Dosa : 43 packets 
+c. Initial stocks of pouches : 39 oil packets 
+d. Initial stocks of panipuri : 43 packets  
+e. Initial stocks of masala�s : 73 packets.
+
+Expectation:
+1) Only one Customer came for shopping, he picked different items with different quantities, 
+print "running out of stock" message if a particular item is sold out 
+otherwise deduct requested quantity from total quantity. Partial selling is allowed.  
+2) Create a method which shows only those item which is "Out of stock".
+3) Create a method which shows only those item which is "Available in stock" & respective quantities.
+ */
+
+public class Assignment7_Shop {
+	int maggieanswer, dosaanswer, pouchesanswer, panipurianswer, masalaanswer;
+
+	void setItemQuantity(int maggie, int dosa, int pouches, int panipuri, int masala) {
+		maggieanswer = maggie;
+		dosaanswer = dosa;
+		pouchesanswer = pouches;
+		panipurianswer = panipuri;
+		masalaanswer = masala;
+	}
+	
+	void inputFromBuyer(int m, int d, int p, int pp, int ml) {
+		if(m>=1){
+			if(m<= maggieanswer) {
+				maggieanswer = maggieanswer-m;
+				System.out.println("Thank you for purchasing "+m+" Qunatity");
+			}else if(m> maggieanswer){
+				System.out.println(+maggieanswer+ " Quantity selled, running out of stock and partial selling is allowed");
+				maggieanswer = 0;
```

Comment:
If request stock is not available mean we are setting direct it zero, 
what if second time your calling same method and let say you request 20 packs, then this looks not logically correct